### PR TITLE
Meeting recording now available

### PR DESCRIPTION
**Pull request by wildgirl**

### CODE REVIEW

**Comment by andrewnicols** [comment]
Hi Helen,

Are we able to use the link directly:

https://moodle.org/mod/bigbluebuttonbn/bbb_view.php?action=play&bn=1&rid=19&rtype=video

**Review by sarjona — DISMISSED** [review]
Helen, thanks for updating the recording for today's session :-)

File: general/community/meetings/202206.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ tags:
 
 ### Details
 
-Tuesday 14 June 2022, at 08:00 UTC ([Check this time in your location](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Moodle+developer+meeting&iso=20220614T08&p1=1440&ah=1))
+Tuesday 14 June 2022 at 08:00 UTC
```
Comment (sarjona) [inline general/community/meetings/202206.md:10]:
```suggestion
Tuesday 14 June 2022 at 08:00 UTC
```

**Review by sarjona — CHANGES_REQUESTED** [review]
Hi Helen! As you'll see, there is an empty space at the end of a line that should be removed :-)

**Review by andrewnicols — APPROVED** [review]

**Comment by wildgirl** [comment]
> Hi Helen,
> 
> Are we able to use the link directly:
> 
> https://moodle.org/mod/bigbluebuttonbn/bbb_view.php?action=play&bn=1&rid=19&rtype=video

Good suggestion Andrew, that would be much better.

**Pull request merged**